fractal_server/config.py

```diff
@@ -114,11 +114,10 @@ def DB_ECHO(self):
     # FRACTAL SPECIFIC
     ###########################################################################
     DATA_DIR_ROOT: str = fail_getenv("DATA_DIR_ROOT")
-    USE_SLURM: bool = bool(int(getenv("USE_SLURM", "0")))
+    PARSL_CONFIG: str = getenv("PARSL_CONFIG", "local")
     PARSL_DEFAULT_EXECUTOR: str = getenv("PARSL_DEFAULT_EXECUTOR", "cpu")
-    if USE_SLURM:
-        SLURM_PARTITION_CPU: str = fail_getenv("SLURM_PARTITION_CPU")
-        # FIXME add gpu partition
+    if PARSL_CONFIG == "custom":
+        PARSL_CONFIG_CUSTOM_FILE: str = fail_getenv("PARSL_CONFIG_CUSTOM_FILE")
```

Comment:
what is this supposed to be? in the prospect of shipping fractal into a container, I would advise against configuration files and keep everything in the environment, which makes it extremely easier to handle in docker / k8s and the like.

in any case It seems like this variable is defined but never used. If that's the case, please remove it until we actually need it